_i18n/de/resources/user-guides/importing_blockchain.md

```diff
@@ -1,62 +1,62 @@
-{% include disclaimer.html translated="no" translationOutdated="no" %}
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
 
-### Purpose and Warning
+### Gegenstand und Warnung
 
-Most people don't need this. To use Monero, just start the software and it will synchronize itself with the peer-to-peer network. Normally, this is much faster than downloading and importing the blockchain as detailed in this guide. This is because you'll be downloading from many peers instead of just a single server, and the Monero daemon will verify each block as it's received, instead of verifying separately after downloading.
+Die meisten Anwender benötigen den Import der Blockchain nicht. Um Monero zu nutzen, musst du lediglich die Software starten, die sich dann mit dem Peer-to-Peer-Netzwerk synchronisiert. In der Regel ist dies deutlich schneller, als würdest du die Blockchain auf die hier beschriebene Weise herunterladen und importieren; du lädst dabei von vielen Peers anstelle von einem einzigen Server herunter, und der Monero-Hintergrunddienst verifiziert dann jeden Block direkt nach dem Eingang, anstelle nach dem Download alles separat zu überprüfen.
 
-This option is mostly useful for development, or possibly if some unusual problem is preventing you from syncing the normal way.
+Die in dieser Anleitung beschriebene Option ist vorwiegend zur Entwicklung von Nutzen, ist aber auch dann hilfreich, wenn dich ein ungewöhnliches Problem vom üblichen Synchronisieren abhalten sollte.
 
-**Never** use the dangerous unverified import option, it is strictly for experts only. Especially, don't use it with any blockchain you download from the Internet, including the official site. It is only safe to use if a) you are importing a file that you exported locally, yourself *and* b) you are absolutely sure it was already fully and properly verified before exporting.
+Verwende **niemals** die gefährliche, ungeprüfte Importoption; diese ist ausschließlich für Experten gedacht. Insbesondere solltest du sie nicht bei einer aus dem Internet, einschließlich von der offiziellen Webseite heruntergeladen Blockchain verwenden. Nur wenn du a) eine Datei importierst, die du eigenhändig lokal exportiert hast *und* b) du absolut sicher bist, dass die Datei vor dem Exportieren gründlich und vollständig überprüft wurde, ist ihr Gebrauch wirklich sicher.
 
-### Step 1
+### Schritt 1
 
-Download the Current bootstrap from https://downloads.getmonero.org/blockchain.raw; you can skip this step if you are importing the blockchain from another source.
+Lade die aktuelle Bootstrap von https://downloads.getmonero.org/blockchain.raw herunter. Wenn du die Blockchain aus einer anderen Quelle importierst, kannst du diesen Schritt überspringen.
```

Comment:
Not sure how to translate "Bootstrap", would try to pick another word for it..

Reply:
Yeah that word is problematic for translators. I hope we will use it less in the future.